packages/cli-tools/test/stream-resend.test.ts

```diff
@@ -24,9 +24,9 @@ describe('resend stream', () => {
             const msg = await stream.publish({ msgId })
             messages.push(msg)
         }
-        await wait(1000)
+        await wait(10000)
```

Comment:
Is this timeout increase needed? And the test timeout change from 20s to 30s?

Reply:
The timeout increases in some tests are added as it can sometimes take a while to recover from network splits when nodes join on empty streams simultaneously.